rest/route/distro.go

```diff
@@ -50,13 +51,16 @@ func (h *distroIDGetSetupHandler) Parse(ctx context.Context, r *http.Request) er
 
 // Run returns the given distro's setup script.
 func (h *distroIDGetSetupHandler) Run(ctx context.Context) gimlet.Responder {
-	d, err := data.FindDistroById(h.distroID)
+	d, err := distro.FindOne(distro.ById(h.distroID))
 	if err != nil {
-		return gimlet.MakeJSONErrorResponder(errors.Wrapf(err, "Database error for find() by distro id '%s'", h.distroID))
+		return gimlet.MakeJSONErrorResponder(errors.Wrapf(gimlet.ErrorResponse{
```

Comment:
Similar to earlier thought, don't need to double wrap the error (the message error is sufficient)